spinn_front_end_common/utilities/utility_objs/executable_type.py

```diff
@@ -0,0 +1,62 @@
+from enum import Enum
+from spinnman.model.enums import CPUState
+
+
+class ExecutableType(Enum):
+    """ supports starting of different types of executables
+    """
+
+    RUNNING = (
+        0,
+        [CPUState.RUNNING],
```

Comment:
This needs a finished as well...

Reply:
yep, dead on. i put them in, then took them out, convincing myself it was safe. but nah ya right

Reply:
rubbered